src/emulator.js

```diff
@@ -87,6 +88,22 @@ export class Emulator {
     this.logProcessor = (item) => item;
 
     return new Promise((resolve, reject) => {
+      let internalId;
+      const checkLiveness = function() {
+        const url = "http://localhost:" + port + "/live"
+        http.get(url, (resp) => {
```

Comment:
I think we can simply use `fetch` here instead of a `http.get` - but this will do as well.
We can refactor it later :)